test/model/recent_senders_checks.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:checks/checks.dart';
+import 'package:zulip/model/recent_senders.dart';
+
+extension RecentSendersChecks on Subject<RecentSenders> {
+  Subject<Map<int, Map<int, MessageIdTracker>>> get streamSenders =>
+    has((rs) => rs.streamSenders, 'streamSenders');
+
+  Subject<Map<int, Map<String, Map<int, MessageIdTracker>>>> get topicSenders =>
+    has((rs) => rs.topicSenders, 'topicSenders');
+
+  Subject<int?> latestMessageIdOfSenderInStream({
+    required int streamId,
+    required int senderId,
+  }) => has(
+          (rs) => rs.latestMessageIdOfSenderInStream(streamId: streamId,
+            senderId: senderId),
+          'latestMessageIdOfSenderInStream',
```

Comment:
Generally we use these "checks" extensions only for getters, not method calls. One reason is that that "name" argument to `has` becomes complicated: just the name of the method is no longer adequate to describe where the problem is, in the error output when the test fails.

Instead, just call the method before passing to `check`, as in `check(model.foo(1, 2)).equals(3)`.